package.json

```diff
@@ -1,9 +1,8 @@
 {
```

Comment:
please undo changes and left this file as it is in main.

After that, knowing that this project still requires some missing points to be considered a release package, I recommend specifying version 0.0.2-beta.1

Reply:
done